### PR TITLE
Cli

### LINKED ISSUE #1: Cli
- `cli` package

### PR DESCRIPTION
**Pull request by estebanprimost**
- Lerna
- Standard
- Husky

Closes: #1

### CODE REVIEW

**Review by dpaez — APPROVED** [review]

**Comment by dpaez** [comment]
Looks good, `husky` setup is missing

File: package.json [hunk 1]
```diff
@@ -22,6 +22,23 @@
...
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
+    }
+  },
+  "semistandard": {
+    "env": [
+      "jest",
+      "node",
+      "browser"
```
Comment (dpaez) [inline package.json]: is it okay to adjust this per module? I can imagine that most of the modules here will run in node and only the dashboard will be used in the browser.

**Comment by dpaez** [comment]
Let's switch back to standard

File: package.json (hunk 1 above)
Comment (estebanprimost) [reply package.json]: Can't make it work with a shared/root config.

**Pull request merged**